packages/client-python/src/rocketride/cli/commands/runtime.py

```
@@ -0,0 +1,888 @@
+
+        print(f'Starting engine v{version} on port {port} (id: {instance_id})...')
+        pid = await spawn_engine(binary, port, instance_id)
+
+    try:
+        log_file = logs_dir(instance_id) / 'stderr.log'
+        await wait_healthy(port, pid=pid, log_file=log_file)
+        # Only persist pid/port once the engine is confirmed healthy
+        async with StateDB() as db:
+            await db.register(instance_id, pid, port, version, 'cli', restart_count=restart_count, desired_state='running')
+        await ensure_autostart_hook()
+        return 0
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)